ci/scripts/alps.sh

```diff
@@ -347,6 +347,91 @@ launch_reframe_1arg() {
         -r
 }
 # }}}
+# {{{ launch_reframe_bencher
+launch_reframe_bencher() {
+    export RFM_AUTODETECT_METHODS="cat /etc/xthostname,hostname"
+    export RFM_USE_LOGIN_SHELL=1
+    # export RFM_AUTODETECT_XTHOSTNAME=1
+    # reframe -V
+    echo "# UENV=$UENV"
+    echo "# img=$img"
+    
+    reframe -C ./config/cscs.py \
+        --report-junit=report.xml \
+        $img \
+        --system=$system \
+        --prefix=$SCRATCH/rfm-$CI_JOB_ID \
+        -c ./checks/microbenchmarks/gpu/amd_gpu/amd_gpu.py \
+        -r
+
+    python3 ./utility/bencher_metric_format.py latest.json
```

Comment:
the lines above could be replaced with:
```
launch_reframe_1arg ...
```

Reply:
the `launch_reframe_bencher` command includes specific tests to run for Bencher.
So, I would prefer to leave it differentiated from the `launch_reframe_1arg `